Client/Frontend/Browser/TabManager.swift

```diff
@@ -67,6 +67,7 @@ class TabManager : NSObject {
         let configuration = WKWebViewConfiguration()
         configuration.processPool = WKProcessPool()
         configuration.preferences.javaScriptCanOpenWindowsAutomatically = !(self.prefs.boolForKey("blockPopups") ?? true)
+        configuration.mediaPlaybackRequiresUserAction = !(self.prefs.boolForKey("allowAutoplay") ?? true)
```

Comment:
The same thing goes for the `mediaPlaybackRequiresUserAction` property. In iOS 9 it's now called `allowsInlineMediaPlayback`. Since you'll need to do this in many places, you can try putting it in an extension on WKWebView to make it less verbose.